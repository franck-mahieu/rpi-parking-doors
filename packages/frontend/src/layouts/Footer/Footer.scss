@import '../../variables';

footer {
  background-color: $header-footer-background-color;
  text-align: center;
  padding: 1rem;
  bottom: 0;
}


#root {
  display: flex;
  min-height: 100vh;
  flex-direction: column;
}

main {
  flex: 1 0 auto;
}
