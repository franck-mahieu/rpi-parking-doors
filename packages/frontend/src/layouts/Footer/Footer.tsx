import React from 'react';

export const Footer = () => {
  return (
    <div className="container grey-text">
      Made with â™¥ by
      <a href="https://twitter.com/Franck_Mahieu"> Franck Mahieu</a>
    </div>
  );
};
