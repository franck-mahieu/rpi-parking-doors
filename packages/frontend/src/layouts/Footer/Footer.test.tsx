import { Footer } from './Footer';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Footer.tsx', () => {
  describe('Footer', () => {
    it('should renders correctly', async () => {
      const component = renderer.create(<Footer />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
