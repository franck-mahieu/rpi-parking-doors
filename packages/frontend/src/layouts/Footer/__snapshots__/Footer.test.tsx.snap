// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.tsx Footer should renders correctly 1`] = `
<div
  className="container grey-text"
>
  Made with â™¥ by
  <a
    href="https://twitter.com/Franck_Mahieu"
  >
     Franck Mahieu
  </a>
</div>
`;
